maths/sum_of_two_positive_numbers_bitwise.py

```diff
@@ -0,0 +1,57 @@
+"""
+Calculates the sum of two positive numbers using bitwise operator
+Wikipedia explanation: https://en.wikipedia.org/wiki/Binary_number
+"""
+
+
+def sum_of_two_positive_numbers_bitwise(number: int, other_number: int) -> int:
```

Comment:
```suggestion
def bitwise_addition_recursive(number: int, other_number: int) -> int:
```
"recursive" because [maths/addition_without_arithmetic.py](https://github.com/TheAlgorithms/Python/blob/1984d9717158c89f9acca2b635a373bad7048633/maths/addition_without_arithmetic.py) already implements this algorithm but iteratively. These two files will have to be merged at some point.